packages/aws-rfdk/lib/deadline/scripts/python/client-rq-connection.py

```diff
@@ -182,17 +190,18 @@ def configure_deadline( config ):
             with open(cert_path, 'wb') as f:
                 f.write(cert_contents)
 
-            call_deadline_command(['SetIniFileSetting', 'ProxySSLCA', cert_path])
             call_deadline_command(['SetIniFileSetting', 'ClientSSLAuthentication', 'Required'])
+
+            repo_args = ['ChangeRepository', 'Proxy', config.render_queue.address]
             repo_args.append(cert_path)
             if config.client_tls_cert_passphrase:
                 passphrase = fetch_secret(config.client_tls_cert_passphrase)
                 repo_args.append(passphrase)
-
-    change_repo_results = call_deadline_command(repo_args)
-    if change_repo_results.startswith('Deadline configuration error:'):
-        print(change_repo_results)
-        raise Exception(change_repo_results)
+            change_repo_results = call_deadline_command(repo_args)
+            if change_repo_results.startswith('Deadline configuration error:'):
```

Comment:
Are we certain that errors in connecting to the repository with always start with this string? Is it possible to capture the exit code of `ChangeRepository` and check that instead?